homeassistant/components/ecobee/weather.py

```diff
@@ -165,10 +165,13 @@ def forecast(self):
             return None
 
         forecasts = []
-        for day in range(1, 5):
+        date = datetime.now()
```

Comment:
The time should be UTC time. Please use our dt util.
https://github.com/home-assistant/core/blob/3881e0cb235437bb93fac81899942e7d854a3889/homeassistant/util/dt.py#L53-L55

Reply:
Thanks. I've change it.